README.md

```diff
@@ -62,6 +63,15 @@ As another example, to run a notebook server, run
 pixi run jupyter lab
 ```
 
+You will likely be running the notebooks in the notebook folder, and may therefore want to create .ipynb versions of the .py scripts. To do this, you can run `jupytext` (see [jupytext docs](https://jupytext.readthedocs.io/en/latest/using-cli.html)) in the environment created with pixi, doing something like:
+```sh
+pixi run jupytext --to notebook .\notebooks\0101_CEDS-prepare.py
+pixi run jupytext --to notebook .\notebooks\0102_GFED4-prepare.py
+pixi run jupytext --to notebook .\notebooks\0103_GFED4-BB4CMIP-prepare.py
+```
```

Comment:
```suggestion
```

I'd recommend not doing this as it scales terribly and you shouldn't need it. Just right-click on the `.py` file and select "open as jupytext" (see attached screenshot).

<img width="1439" alt="Screenshot 2024-10-22 at 5 20 03 pm" src="https://github.com/user-attachments/assets/f4bc5326-5402-4b43-9a8b-1753245228b8">

Reply:
Addressed in [b2b0209](https://github.com/iiasa/emissions_harmonization_historical/pull/18/commits/b2b0209639c59efb6e62fb2f3b0194f9c5bbbe77)